message-bus-helper-hapi-proxy/swarm.sh

```diff
@@ -3,7 +3,7 @@
 declare ACTION=""
 declare COMPOSE_FILE_PATH=""
 declare UTILS_PATH=""
-declare service_name=""
+declare SERVICE_NAMES=""
```

Comment:
Let's always use an array to avoid confusion. Thus allowing to add other services when needed.